index.ts

```diff
@@ -0,0 +1,60 @@
+const products = require("./data/products.json");
```

Comment:
use `import` and explain the difference between require and import

Reply:
import is the keyword for es6 (esmodules) import, and require is the keyword for commonjs modules import. in this case we are importing a JSON file and now its done by this way (this line 1, line 58 and the compilerOptions in the tsconfig.json file).